packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -539,6 +555,36 @@ contract Deploy is Deployer {
         addr_ = address(portal);
     }
 
+    /// @notice Deploy the OptimismPortal2
+    function deployOptimismPortal2() public broadcast returns (address addr_) {
+        console.log("Deploying OptimismPortal2 implementation");
+
+        // Could also verify this inside DeployConfig but doing it here is a bit more reliable.
+        require(
+            uint32(cfg.respectedGameType()) == cfg.respectedGameType(), "Deploy: respectedGameType must fit into uint32"
+        );
+
+        OptimismPortal2 portal = new OptimismPortal2{ salt: _implSalt() }(
+            cfg.proofMaturityDelaySeconds(),
```

Comment:
nit: prefer kwargs style syntax when 2 or more args to ensure that they are correct

Reply:
SGTM